modules/scripts/startup-script/files/running-script-warning.sh

```diff
@@ -13,17 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-WARNING=$(
-	cat <<EOF
-** NOTICE **: System services may not be running until startup scripts complete.
-The output of the command below will end with "Finished Google Compute Engine 
-Startup Scripts." when they are complete. Please review the output for any 
-errors which may indicate that services are unhealthy. 
-
-sudo journalctl -b 0 -u google-startup-scripts.service
-EOF
+SCRIPT_COMPLETE_FILE="/run/startup_script_msg"
+WARNING_MESSAGE=$(
+	cat <<-EOF
+		** NOTICE **: HPC Toolkit startup scripts have not started yet.
+		Systems may not be configured or running.
+	EOF
 )
 
-echo
-echo "${WARNING}"
-echo
+if [ -f "${SCRIPT_COMPLETE_FILE}" ]; then
+	echo
```

Comment:
These two echos are creating blank lines when the startup message is empty so that the prompt does not appear at the top of a fresh terminal. Please make it appear at the top (i.e. "normal").

I suggest you do this by incorporating any desired whitespace into the actual message file and ensuring that you do not add any additional newlines.

Reply:
I saw this and was fixing it.  My alternate solution was to check if the file was empty and if it is, just don't print anything.

Reply:
I considered that too. Do anything simple and robust that resolves it.

Reply:
Done